share/dkg/dkg.go

```diff
@@ -591,8 +591,30 @@ func (d *DistKeyGenerator) ProcessResponses(bundles []*ResponseBundle) (*Result,
 			if response.Status == Complaint {
 				foundComplaint = true
 			}
+			validAuthors = append(validAuthors, bundle.ShareIndex)
 		}
 	}
+
+	// In case of fast sync, we want to make sure all share holders have sent a
+	// valid response (success or complaint). All share holders that did not
+	// will be evicted from the final group. Since we are using a broadcast
+	// channel, if a node is honest, its response will be received by all honest
+	// nodes.
+	if d.c.FastSync {
+		// we only need to look at the nodes that did not sent any response,
+		// since the invalid one are already markes as evicted
+		allSent := append(validAuthors, d.evictedHolders...)
+		isPresent := isEvicted
```

Comment:
this is counter-intuitive. from what i can see, this method is only used in this if block. can you directly call 'isEvicted' in the loop below. Renaming it to a variable named the inverse property makes it more confusing what is being checked.